code_schemes/dadaab_ws_correct_dataset.json

```diff
@@ -0,0 +1,354 @@
+      "StringValue": "dadaab_s01e04",
+      "NumericValue": 8,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01e04"
+      ]
+    },
+    {
+      "CodeID": "code-401b6cc9",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab s01e05",
+      "StringValue": "dadaab_s01e05",
+      "NumericValue": 9,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01e05"
+      ]
+    },
+    {
+      "CodeID": "code-afc32d63",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab s01e06",
+      "StringValue": "dadaab_s01e06",
+      "NumericValue": 10,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01e06"
+      ]
+    },
+    {
+      "CodeID": "code-1b262084",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab s01e07",
+      "StringValue": "dadaab_s01e07",
+      "NumericValue": 11,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01e07"
+      ]
+    },
+    {
+      "CodeID": "code-3bf620g4",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab s01 intro",
+      "StringValue": "dadaab_s01_intro",
+      "NumericValue": 12,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab s01 intro"
+
+      ]
+    },
+    {
+      "CodeID": "code-62e86a77",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab girls education champions",
+      "StringValue": "dadaab_girls_education_champions",
+      "NumericValue": 13,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab girls education champions"
+      ]
+    },
+    {
+      "CodeID": "code-e24f1e4a",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab encouragement for boys",
+      "StringValue": "dadaab_encouragement_for_boys",
+      "NumericValue": 14,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab encouragement for boys"
+      ]
+    },
+    {
+      "CodeID": "code-5c606074",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab unmarried fathers community view",
+      "StringValue": "dadaab_unmarried_fathers_community_view",
+      "NumericValue": 15,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab unmarried fathers community view"
+      ]
+    },
+    {
+      "CodeID": "code-90616cb5",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab lessons learnt",
+      "StringValue": "dadaab_lessons_learnt",
+      "NumericValue": 16,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab lessons learnt"
+      ]
+    },
+    {
+      "CodeID": "code-e3c25a49",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab show suggestions",
+      "StringValue": "dadaab_show_suggestions",
+      "NumericValue": 17,
+      "VisibleInCoda": true,
+      "MatchValues": [
+        "dadaab show suggestions"
+         ]
+    },
+    {
+      "CodeID": "code-bdd2b90b",
+      "CodeType": "Normal",
+      "DisplayText": "dadaab show suggestions",
```

Comment:
Duplicated code

Reply:
thanks for the catch removed.

Reply:
I just released https://github.com/AfricasVoices/CoreDataModules/releases/tag/v0.14.1, which will automatically identify duplicates when you try to run the pipeline. Sorry for not doing so earlier, I don't think there was any reason to hold that up.